src/manifest-converter.ts

```diff
@@ -80,25 +97,49 @@ export function generatePackageJson(
     license: bowerJson.license,
     homepage: bowerJson.homepage,
     dependencies: <any>{},
-    devDependencies: <any>{}
+    devDependencies: <any>{},
+    // resolutions needed for wct-browser-legacy
+    resolutions: {
```

Comment:
Can this go in a configuration somewhere? Could still be in code, but in a more top-level location?

Also, this is unfortunate... will users get a warning if we're overriding their versions on a conflict?